apps/src/constants.js

```diff
@@ -44,6 +44,7 @@ exports.TestResults = {
 
   // start using negative values, since we consider >= 20 to be "solved"
   EMPTY_FUNCTION_NAME: -1,       // We have a variable or function with the name ""
+  NESTED_FOR_SAME_VARIABLE: -2,
```

Comment:
Actually, wouldn't it be pretty non-risky net-positive to separate them out now? Since we'll know that -1 meant "either or" before we separated them, and we'll start getting clarity once separated. 

That said, the only real use case is data reporting on our end, but who knows at some point we might want to graph error instances to debug issues